spec/money_helper_spec.rb

```diff
@@ -37,7 +200,6 @@
       expect(MoneyHelper.money_to_text(30_000, 'SRD')).to eql('SRD $30,000')
       expect(MoneyHelper.money_to_text(30_000, 'TWD')).to eql('TWD $30,000')
       expect(MoneyHelper.money_to_text(30_000, 'TTD')).to eql('TTD $30,000')
-      expect(MoneyHelper.money_to_text(30_000, 'UYU')).to eql('UYU $30.000')
```

Comment:
By [updating the version](https://github.com/artsy/money_helper/pull/20/files#r710143565), some specs are already broken.